client/assets/css/james.css

```diff
@@ -0,0 +1,25 @@
+h2 {
+    font-family: 'Oswald', sans-serif;
+    text-align: center;
+}
+body {
+    color: cornsilk;
+    text-align: left;
+    background: url(../images/Edited\ beach\ milky\ way.jpg) no-repeat center center fixed;
+    -webkit-background-size: cover;
+    -moz-background-size: cover;
+    -o-background-size: cover;
+
+}
+.card {
+    background: transparent;
+}
+.card-body {
+    text-align: center;
```

Comment:
Since you are using `text-align: center;` on 4 different elements here, I would recommend creating a utility class that you can apply to all of them to not repeat yourself.

Reply:
Since you're using bootstrap, you can also just put a `class="text-center"` on those elements

Reply:
https://getbootstrap.com/docs/4.0/utilities/text/